docs/cardinal/game/world/api-reference.mdx

```diff
@@ -130,27 +166,59 @@ func main() {
 	// 1. MoveSystem
 	// 2. HealthRegenSystem
 	// 3. AttackSystem
-	cardinal.RegisterSystems(world, systems.MoveSystem, systems.HealthRegenSystem, systems.AttackSystem)
+	cardinal.RegisterSystems(world,
+        systems.MoveSystem,
+        systems.HealthRegenSystem,
+        systems.AttackSystem)
```

Comment:
i think these should be indented over a bit

Reply:
done